packages/mobile/src/components/now-playing-drawer/AudioControls.tsx

```diff
@@ -32,6 +42,18 @@ const createStyles = () =>
       display: 'flex',
       alignItems: 'center'
     },
+    emptyPlaceholder: {
+      minHeight: 24,
```

Comment:
spacing for all these?

Reply:
oh looks like this is a createStyles legacy thing, would you mind updating to useStyles format?